cost-analyzer/values.yaml

```diff
@@ -424,6 +424,7 @@ sigV4Proxy:
 kubecostModel:
   image: "gcr.io/kubecost1/cost-model"
   imagePullPolicy: Always
+  # openSourceOnly: false
```

Comment:
What does this do? Can you add a simple comment? I personally would like to know as would users.

Reply:
setting to true uses `quay.io/kubecost1/kubecost-cost-model` as the base repository of the image here. I added a comment saying as much